src/test/java/de/retest/recheck/cli/subcommands/IgnoreIT.java

```diff
@@ -65,10 +67,10 @@ public void ignore_should_not_ignore_the_report_because_there_are_no_differences
 
 	@Test
 	public void ignore_should_find_differences_and_update_the_recheck_ignore_file() throws Exception {
-		System.setProperty( ProjectConfiguration.RETEST_PROJECT_ROOT, temp.getRoot().getAbsolutePath() );
-		temp.newFolder( "src", "main", "java" );
-		temp.newFolder( "src", "test", "java" );
-		temp.newFolder( ".retest" );
+		System.setProperty( ProjectConfiguration.RETEST_PROJECT_ROOT, temp.getRoot().toString() );
+		Files.createDirectories( temp.getRoot().toPath().resolve( "src/main/java" ) );
```

Comment:
AFAIK `Files.createDirectories( ... )` is only needed in JUnit 5 tests, where the `TempDirectory` extension only provides a plain `Path`. But the `TemporaryFolder` rule in JUnit 4 already has facilities for this, such as the removed `newFolder( ... )`. Or do I miss something?